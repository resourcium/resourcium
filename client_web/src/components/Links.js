import React from 'react';
import './Links.scss';
import { USEFUL_LINKS } from 'src/common/config';

export default function Other() {
    /*var today = new Date();
    var date_today = today.getFullYear() + '-' + today.getMonth() + '-' + today.getDay();
    var time_today = today.getHours()+':'+today.getMinutes()+':'+today.getSeconds();

    var tomorrow = new Date();
    tomorrow.setDate(tomorrow.getDate() + 1);
    var date_tom = tomorrow.getFullYear() + '-' + tomorrow.getMonth()+ '-' + tomorrow.getDay();
    */
    return (
        <div className="other">
            <p>Here are some useful links within and outside of the university that you may want to engage with.</p>
            {USEFUL_LINKS.map((link, i) => (
                <a href={link.href} key={i} target="_blank" rel="noreferrer">
                    {link.label}
                </a>
            ))}
        </div>
    );
}
