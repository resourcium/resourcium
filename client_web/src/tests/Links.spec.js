import { create } from 'react-test-renderer';
import React from 'react';

import Links from 'src/components/Links';

import { wrapProvider } from 'src/tests/util';

test('links snapshot matches', () => {
    expect(create(wrapProvider(<Links />))).toMatchSnapshot();
});
