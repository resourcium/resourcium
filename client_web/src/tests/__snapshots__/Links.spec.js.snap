// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`links snapshot matches 1`] = `
<div
  className="other"
>
  <p>
    Here are some useful links within and outside of the university that you may want to engage with.
  </p>
  <a
    href="https://www.ucl.ac.uk/library/"
    rel="noreferrer"
    target="_blank"
  >
    Library
  </a>
  <a
    href="https://www.meetup.com/home/"
    rel="noreferrer"
    target="_blank"
  >
    Meetup
  </a>
  <a
    href="https://www.ucl.ac.uk/students/support-and-wellbeing/wellbeing"
    rel="noreferrer"
    target="_blank"
  >
    Mental Wellbeing
  </a>
  <a
    href="https://studentsunionucl.org/"
    rel="noreferrer"
    target="_blank"
  >
    Student Union
  </a>
</div>
`;
